tests/rptest/services/kgo_verifier_services.py

```diff
@@ -163,10 +159,10 @@ def wait_node(self, node, timeout_sec=None):
         # Let the worker fall through to the end of its current iteration
         self.logger.debug(
             f"wait_node {self.who_am_i()}: waiting for worker to complete")
-        wait_until(lambda: self._status.active is False or self._status_thread.
-                   errored,
-                   timeout_sec=timeout_sec,
-                   backoff_sec=5)
+        self._redpanda.wait_until(lambda: self._status.active is False or self.
```

Comment:
should we make it configurable, some of the tests may inject failures and wait when one of the nodes is down

Reply:
I was thinking about that: for the moment, we can just avoid using self.redpanda.wait_until in those types of test, but that's a bit fragile.

It feels like we need a more general mechanism for the failure injectors to indicate which nodes are supposed to be online: currently they don't update the _started list, but perhaps they could?  or have another list specfically for failure injectors to register which nodes they are stopping temporarily